### ISSUE TITLE
About the pre-trained model

### ISSUE THREAD

**Issue by bobo0810** [cavalleria/cavaface.pytorch#33]
Thank you  for this work. I want to use the pre-trained model of AttentionNet-IRSE-92 from the MODEL_ZOO.md for fine-tuning. Where can I get the pre-trained model?

**Comment by cavalleria** [comment]
The AttentionNet-IRSE pretrained model is not avaiable. And I saved the model use `torch.jit.save` api which seem like not support for finetuning. You can retrain it use this to save model.[https://github.com/cavalleria/cavaface.pytorch/blob/13182ecc349ca050fa5a877045390a41037313a7/train.py#L352](url)

**Comment by bobo0810** [comment]
> The AttentionNet-IRSE pretrained model is not avaiable. And I saved the model use `torch.jit.save` api which seem like not support for finetuning. You can retrain it use this to save model.[https://github.com/cavalleria/cavaface.pytorch/blob/13182ecc349ca050fa5a877045390a41037313a7/train.py#L352](url)

OK，thanks!

**Comment by cavalleria** [comment]
The models from Model Zoo.md is saved using api `torch.jit.save` which can evaluate it without defining model python files, but it seem like cannot finetune.😂

**Comment by xsacha** [comment]
You can load the weights from torch.jit.load and set these in to the state_dict of a real model (from BACKBONE_DICT) in order to support finetuning.

**Comment by bobo0810** [comment]
> You can load the weights from torch.jit.load and set these in to the state_dict of a real model (from BACKBONE_DICT) in order to support finetuning.

Oh, that's great. Can you provide the pre-trained model of AttentionNet-IRSE-56/92 so that I can fine-tune it? Thank you @cavalleria @xsacha

**Comment by xsacha** [comment]
@bobo0810 
Should be like this:

```
mymodel = AttNet96()
mymodel.load_state_dict(torch.jit.load('checkpoint.pt').state_dict())
```

**Comment by FelixZhang7** [comment]
@cavalleria hello，according to data augmentation result in model_zoo.md，baseline is the best, so we do not need to do any  data augmentation?

**Comment by xsacha** [comment]
> @cavalleria hello，according to data augmentation result in model_zoo.md，baseline is the best, so we do not need to do any data augmentation?

When trained for the same amount of time, from scratch, baseline is better. This will be the case for any sort of classifier AFAIK.

Augmentation requires either a fine-tune on a baseline or longer training to see optimal results.

**Comment by bobo0810** [comment]
> > @cavalleria hello，according to data augmentation result in model_zoo.md，baseline is the best, so we do not need to do any data augmentation?
> 
> When trained for the same amount of time, from scratch, baseline is better. This will be the case for any sort of classifier AFAIK.
> 
> Augmentation requires either a fine-tune on a baseline or longer training to see optimal results.

@FelixZhang7 Although model.md is like this, my experimental experience is that adding data augmentation will improve performance.
@xsacha In order to make the network converge better, data augmentation should be performed on the pre-trained model and continue training

**Comment by FelixZhang7** [comment]
@xsacha @cavalleria @bobo0810 Hello, I want to find a network with a better accuracy than resnet50-IR, and it is better to have similar inference speed，could you give me some advice ?

**Comment by xsacha** [comment]
@FelixZhang7 AttNet-56

**Comment by bobo0810** [comment]
> @xsacha @cavalleria @bobo0810 Hello, I want to find a network with a better accuracy than resnet50-IR, and it is better to have similar inference speed，could you give me some advice ?

I haven't compared the speed of the networks yet. Refer to modelzoo.md, I am using attention_irse.

**Comment by cavalleria** [comment]
@bobo0810 efficient-b1/mobilenetv3(enlarge width) and so on.

**Comment by FelixZhang7** [comment]
@xsacha I‘ve tried AttNet-56-IR，almost the same acc compare with resnet50-IR...
